src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -959,9 +959,11 @@ type AdaptiveFSharpLspServer(workspaceLoader: IWorkspaceLoader, lspClient: FShar
         let! ct = Async.CancellationToken
         do! semaphore.WaitAsync(ct) |> Async.AwaitTask
 
-        use _ = new ProgressListener(lspClient)
+        use pgl = new ProgressListener(lspClient)
         use progressReport = new ServerProgressReport(lspClient)
 
+        use _ = ct.Register(fun () -> pgl.Dispose())
```

Comment:
I've noticed this notification sometimes sticking around so trying to add additional dispose logic.